tools/custom-action/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:23.04
+
+ENV LANG=en_US.UTF-8
+
+RUN \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt update -y && \
+  apt install -y locales && \
+  locale-gen en_US.UTF-8 && \
+  sysctl -w kernel.unprivileged_userns_clone=1 && \
+  apt install -y \
+    nodejs \
+    npm \
+    git \
+    sudo \
+    libgtk-3-dev \
+    libnotify-dev \
+    libgconf-2-4 \
+    libnss3 \
+    libxss1 \
+    libasound2 \
+    python3-full \
+    python3-pip \
+    rsync
+
+RUN \
+  export DEBIAN_FRONTEND=noninteractive && \
```

Comment:
DEBIAN_FRONTEND seems like it's only for `apt`/`apt-get` and I don't think we need it here or below.

Reply:
Cleared!